python/py-tensorflow_macos/Portfile

```diff
@@ -1,114 +0,0 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-
-PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           python 1.0
-
-github.setup        apple tensorflow_macos 0.1 v alpha3
```

Comment:
Understood. Because this is all still in the alpha stage, and because many people, including myself, are [observing](https://developer.apple.com/forums/thread/687654) subpar performance with Tensorflow's version  `tensorflow-macos` , I think we should keep Apple's version `tensorflow_macos` around for now. I added commented-out obsolescence code for when that happens, which I hope is the next TF version.

I also think we really need to be consistent with upstream naming (even for small stuff like `_` versus `-`) because otherwise it becomes very confusing about which checkpoint is being used and which benchmarks to expect. Especially when the benchmarks of these early alpha versions are quite divergent.